pkg/hive/health/command.go

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package health
+
+import (
+	"fmt"
+	"os"
+	"slices"
+	"strings"
+
+	healthPkg "github.com/cilium/cilium/pkg/health/client"
+	"github.com/cilium/cilium/pkg/hive/health/types"
+	"github.com/cilium/hive"
+	"github.com/cilium/hive/script"
+	"github.com/cilium/statedb"
+
+	"github.com/spf13/pflag"
+)
+
+func healthCommands(db *statedb.DB, table statedb.Table[types.Status]) hive.ScriptCmdsOut {
+	return hive.NewScriptCmds(map[string]script.Cmd{
+		"health":    healthTreeCommand(db, table),
+		"health/ok": allOK(db, table),
+	})
+}
+
+func healthTreeCommand(db *statedb.DB, table statedb.Table[types.Status]) script.Cmd {
+	return script.Command(
+		script.CmdUsage{
+			Summary: "List registered metrics",
+			Args:    "[reporter-id-prefix]",
+			Flags: func(fs *pflag.FlagSet) {
+				fs.StringP("match", "m", "", "Output only health reports where the reporter ID path contains the substring")
+				fs.StringArrayP("levels", "s", []string{types.LevelOK, types.LevelDegraded, types.LevelDegraded},
+					"Output only health reports with the specified state (i.e. ok,degraded,stopped)")
+				fs.StringP("file", "f", "", "File to write output to")
```

Comment:
Other commands are using `-o`/`--output` for setting the output file and using `-f`/`--format` for the output format.